crates/data_components/src/delta_lake.rs

```diff
@@ -261,8 +266,16 @@ impl DeltaTable {
             .with_parquet_file_reader_factory(Arc::clone(parquet_file_reader_factory))
             .with_predicate(Arc::clone(physical_expr));
 
+        // Matches keying used by `ObjectStoreRegistry::get_url_key`
+        let object_store_url = ObjectStoreUrl::parse(format!(
```

Comment:
Make the Delta `ObjectStoreUrl` bindable across ser/de gap by specifying the base URL/scheme